### PR TITLE
{tools}[foss/2018b] cdsapi v0.1.4 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3135.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/cc602bff0ee0342376414f48809ff028 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2120.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5586ddfbfea821f15ec4243cbc7a8ac3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2609.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c4557822d29eb7fcc4b535b0d55ef6b6 for a full test report.

File: easybuild/easyconfigs/c/cdsapi/cdsapi-0.1.4-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'cdsapi'
+version = '0.1.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/wheel'
+description = """A built-package format for Python."""
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/cdsapi/cdsapi-0.1.4-foss-2018b-Python-3.6.6.eb]: yes, wheel is indeed a built-package format for Python... ;)

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/2890ea29dfd7be13826e6e9eba48a2f5 for a full test report.

File: easybuild/easyconfigs/c/cdsapi/cdsapi-0.1.4-foss-2018b-Python-3.6.6.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/c/cdsapi/cdsapi-0.1.4-foss-2018b-Python-3.6.6.eb]: @boegel ping on fixing homepage and descr for this one...

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/4be4959a9b0494a31d05799dc618af48 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**